docs/tutorials/runtime-module-in-own-crate.md

```diff
@@ -0,0 +1,133 @@
+
+Go ahead and compile your new crate with `cargo build`. It won't compile yet, because that template module has some dependencies from substrate itself. The compiler errors give you clues about what you need to include. We can just copy the relevant dependencies from the original node template runtime's `Cargo.toml`. Add these lines to your crate's `Cargo.toml`.
+
+```toml
+[features]
+default = ['std']
+std = [
+    'parity-codec/std',
+    'support/std',
+    'system/std',
+]
+
+[dependencies.parity-codec]
+default-features = false
+features = ['derive']
+version = '3.5'
+
+[dependencies.support]
+default_features = false
+git = 'https://github.com/paritytech/substrate.git'
+package = 'srml-support'
+rev = 'bb68456966bf9d767651773dbaadd6787bce1884'
+
+[dependencies.system]
+default_features = false
+git = 'https://github.com/paritytech/substrate.git'
+package = 'srml-system'
+rev = 'bb68456966bf9d767651773dbaadd6787bce1884'
+```
+
+At this point we have a runtime module that compiles. It comes with a test, so let's see if that test passes .
+```bash
+cargo test
+```
+
+Turns out we'll need some additional dependencies to make the test pass. You can see this from the error messages, or by looking at the test module itself. Let's add `primitives`, `runtime-primitives`, and `runtime-io` as dev dependencies. The dev dependencies do not need to be listed as part of `std`. TODO is that even proper phrasing?
+
+Add these lines to your `Cargo.toml`.
+```toml
+
+[dev-dependencies.primitives]
+default_features = false
+git = 'https://github.com/paritytech/substrate.git'
+package = 'substrate-primitives'
+rev = 'bb68456966bf9d767651773dbaadd6787bce1884'
+
+[dev-dependencies.runtime-primitives]
+default_features = false
+git = 'https://github.com/paritytech/substrate.git'
+package = 'sr-primitives'
+rev = 'bb68456966bf9d767651773dbaadd6787bce1884'
+
+[dev-dependencies.runtime-io]
+default_features = false
+git = 'https://github.com/paritytech/substrate.git'
+package = 'sr-io'
+rev = 'bb68456966bf9d767651773dbaadd6787bce1884'
+```
+
+We're now good to go for this particular module. It will take a little experimentation to gain familiarity with what dependencies you need for each module you dream up. So the take-away from this section is not that `Cargo.toml` should always look like it did today. Rather, `Cargo.toml` needs to have the correct dependencies and you now know a few places to look for how to specify them when you get stuck.
+
+## Add your Runtime Module to your Node
+With our runtime module now compiling and passing tests (well, passing test anyway) we're ready to add it to our node template.
+
+First we need to add our newly-created crate as a dependency in the node runtime's `Cargo.toml`.
+
+```toml
+[dependencies.test_module]
+default_features = false
+path = "../../test_module"
+# Optionally specify a version
+# version = "0.1.0"
+```
+
+And remember to update the `std` section/
+```toml
```

Comment:
the toml tag is doing something weird

Reply:
This is what I see after `yarn start`. Did you see something different?
![image](https://user-images.githubusercontent.com/2915325/60048379-02589300-969a-11e9-8ee5-96004751e75e.png)

Maybe it was related to that specific short snippet?